NetKAN/KSPScienceRedone.netkan

```diff
@@ -0,0 +1,17 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "KSPScienceRedone",
+    "$kref"        : "#/ckan/github/ValynEritai/KSPScienceRedone",
```

Comment:
This needs to match the GitHub Repository name:

    "$kref"        : "#/ckan/github/ValynEritai/ksp-science-redone",